pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -212,6 +253,11 @@ async def raise_if_pending_jobs(self) -> None:
     def aet(self) -> str:
         return str(config("ORTHANC_RAW_AE_TITLE"))
 
-    async def send_existing_study_to_anon(self, resource_id: str) -> Any:
+    async def send_study_to_anon(self, resource_id: str) -> str:
         """Send study to orthanc anon."""
-        return await self._post("/send-to-anon", data={"ResourceId": resource_id})
+        reponse = await self._post(
+            "/send-to-anon",
+            data={"ResourceId": resource_id, "Synchronous": False, "Timeout": self.dicom_timeout},
+            timeout=self.http_timeout,
+        )
+        return str(reponse["ID"])
```

Comment:
Ah this is a custom endpoint, so we'd need to update the `pixl.py` plugin to use the timeout value when sending and do it async. Worth trying out to see if the OnChange callback only sending if the study is stable and the project name is set is required for the study to be kept in orthanc raw first. If it is then go back to using that I guess, and then updating the send to be asynchronous

Reply:
marking as resolved as we've moved `send_study_to_anon` out of the plugin and into `pixl_imaging._processing`